samza-core/src/main/java/org/apache/samza/task/AsyncRunLoop.java

```diff
@@ -302,7 +304,7 @@ private boolean markProcessed() {
     COMMIT,
     PROCESS,
     END_OF_STREAM,
-    NO_OP
+    TIMER, NO_OP
```

Comment:
Why do we add TIMER, if the actual execution code is exactly the same as WINDOW? I thought that the initial implementation would just be triggering the timers via WINDOW calls?